app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -33,68 +33,68 @@ keywords:
 
-Email your chosen provider to check they can sponsor you, then [apply for your teacher training course](/how-to-apply-for-teacher-training). 
+Email your chosen provider to check they can sponsor you, then [apply for your teacher training course](/how-to-apply-for-teacher-training).
 
-When you have a confirmed offer of a training place, you’ll be able to apply for your visa.  
+When you have a confirmed offer of a training place, you’ll be able to apply for your visa.
 
-For a [Student visa](https://www.gov.uk/student-visa), you will also need to: 
+For a [Student visa](https://www.gov.uk/student-visa), you will also need to:
 
 * have a reference number, called a confirmation of acceptance for studies (CAS), which your training provider will give you
 * [have enough money to support yourself and pay for your course](https://www.gov.uk/student-visa/money)
 * [meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/student-visa/knowledge-of-english)
 
-For a [Skilled Worker visa](https://www.gov.uk/skilled-worker-visa), you will also need to: 
+For a [Skilled Worker visa](https://www.gov.uk/skilled-worker-visa), you will also need to:
 
-* have a ‘certificate of sponsorship’ from the school which is paying you, confirming they will sponsor you 
+* have a ‘certificate of sponsorship’ from the school which is paying you, confirming they will sponsor you
 * [meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
-* check your salary meets the requirement for a Skilled worker visa 
+* check your salary meets the requirement for a Skilled worker visa
 
-### Salary requirement for a Skilled Worker visa 
+### Salary requirement for a Skilled Worker visa
 
-Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a Skilled Worker visa – this varies by region, as follows: 
+Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a Skilled Worker visa – this varies by region, as follows:
 
-* inner London: £25,831 
+* inner London: $salaries_skilledworkervisatraineeminfulltime_innerlondon$
 
-* outer London: £24,415 
+* outer London: $salaries_skilledworkervisatraineeminfulltime_outerlondon$
 
-* London fringe: £23,200 
+* London fringe: $salaries_skilledworkervisatraineeminfulltime_londonfringe$
 
-* rest of England: £23,200 
+* rest of England: $salaries_skilledworkervisatraineeminfulltime_restofengland$
 
-Check the region you will be working in with your teacher training provider. 
+Check the region you will be working in with your teacher training provider.
 
-You may be able to train part-time, but your part-time salary must be at least £23,200.
+You may be able to train part-time, but your part-time salary must be at least $salaries_skilledworkervisatraineeminparttime$.
```

Comment:
This value isn't showing up on the review page, but can't work out why!

Reply:
I think it would need `_restofengland` on the end

Reply:
resolving as changing the format of this variable